examples/hive/sampleapp/populate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash 
+
+HIVE_CLI_PORT=`kubectl get svc hiveserver -n default -o jsonpath='{.spec.ports[?(@.name=="cliservice")].nodePort}'`
+
+HIVE_CLI_IP=`minikube service hiveserver --url -n default | awk -F':' -v port="$HIVE_CLI_PORT" '{if ($3 == port) print $2}' - | cut -d / -f 3`
+
+echo $HIVE_CLI_IP:$HIVE_CLI_PORT
+
+DOCKER_REGISTRY_COMPONENTS="registry_used_in_installation"
+HIVESERVER_IMAGE="hive-server:latest"
+
+docker run -v $(PWD):/sampleapp -it ${DOCKER_REGISTRY_COMPONENTS}/${HIVESERVER_IMAGE} bin/beeline -u "jdbc:hive2://$HIVE_CLI_IP:$HIVE_CLI_PORT/;transportMode=http;httpPath=/cliservice" -f /sampleapp/sample.hql
```

Comment:
$(PWD) -> ${PWD}

Reply:
Also, this last command returns:
```
20/03/13 09:26:22 [main]: WARN jdbc.HiveConnection: Failed to connect to 192.168.39.51:31321
Error: Could not open client transport with JDBC Uri: jdbc:hive2://192.168.39.51:31321/;transportMode=http;httpPath=/cliservice: Could not establish connection to jdbc:hive2://192.168.39.51:31321/;transportMode=http;httpPath=/cliservice: org.apache.http.conn.HttpHostConnectException: Connect to 192.168.39.51:31321 [/192.168.39.51] failed: Connection refused (Connection refused) (state=08S01,code=0)

```

Reply:
Variable substitution fixed in https://github.com/srikumar003/dataset-lifecycle-framework/commit/74d2c763bc1bdb3190430c2ea89f60c7c6cfc999

Provided a convenience script for setting up sample app